packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/features/listFeatures.ts

```diff
@@ -1,31 +1,33 @@
 import getAutoBulletListStyle from '../utils/getAutoBulletListStyle';
 import getAutoNumberingListStyle from '../utils/getAutoNumberingListStyle';
-import {
-    blockFormat,
-    commitListChains,
-    setIndentation,
-    toggleBullet,
-    toggleNumbering,
-    toggleListType,
-} from 'roosterjs-editor-api';
 import {
     Browser,
+    cacheGetEventData,
```

Comment:
Is this order change generated by some extension? It looks nice, please consider add the extension to `extensions.json`

Reply:
I think it was done with miclo.sort-typescript-imports which is already in the file. Not quite sure why it wasnt ordered before.

Reply:
I see. So let's just leave it there.